.github/workflows/docsite-publish-ghpages.yml

```diff
@@ -0,0 +1,73 @@
+# Workflow name
+name: 'Docsite publish'
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.head_commit.message, 'applying package updates') || github.event_name == 'workflow_dispatch' }}
+
+    outputs:
+      status: ${{ steps.verify-react-components-changed.outputs.any_changed == 'true' || github.event_name == 'workflow_dispatch' }}
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout [master]
+
+      - name: Verify react-compoenents has changed
```

Comment:
is it possible to force this job to run even if components haven't changed? How do I push new docs, or simply a typo fix into the website?

Reply:
There's a manual trigger option for this. You can always trigger manual deployments because of this condition: `${{ steps.verify-react-components-changed.outputs.any_changed == 'true' || github.event_name == 'workflow_dispatch' }}`